### PR TITLE
app-emacs/projectile: Added optfeature message to inform users on other

### PR DESCRIPTION
**Pull request by lima-limon-inc**
packages to improve projectile's functionality.
Source: https://docs.projectile.mx/projectile/usage.html

### CODE REVIEW

File: app-emacs/projectile/projectile-2.8.0-r1.ebuild [hunk 1]
```diff
@@ -25,3 +25,11 @@ src_test() {
 
 	elisp-test
 }
+
+pkg_postinst() {
+	# Descriptions for this packages' purpose were taken from Projectile's
+	# home page https://docs.projectile.mx/projectile/usage.html
+	optfeature_header "Performance can be gained in ${PN} by installing the following packages:"
```
Comment (ulm) [inline app-emacs/projectile/projectile-2.8.0-r1.ebuild:32]: Please limit line length to 80 positions.
Policy reference: https://devmanual.gentoo.org/ebuild-writing/file-format/index.html#indenting-and-whitespace

Comment (lima-limon-inc) [reply app-emacs/projectile/projectile-2.8.0-r1.ebuild]: Noted! Thanks for the feedback. 
I changed it in the new commit

**Review by ulm — APPROVED** [review]

**Comment by lima-limon-inc** [comment]
Thank you @ulm!
Quick question, do you know why Github does not show your profile picture in the last commit?
Did I write the "Suggested-by" tag wrong?

**Comment by ulm** [comment]
> Thank you @ulm! Quick question, do you know why Github does not show your profile picture in the last commit? Did I write the "Suggested-by" tag wrong?

IIUC the picture will only be shown for the author and for the committer.

**Comment by Flowdalic** [comment]
FWIW, I believe GitHub also uses the Co-authored-by tag to cross reference commits with GitHub profiles.

**Comment by ulm** [comment]
Is it important? :man_shrugging:
In fact, I think that I don't deserve much credit here, so feel free to squash the commits into one and drop the Suggested-by.

**Comment by Flowdalic** [comment]
> Is it important? 🤷‍♂️

Credit when credit is due. :) In FOSS you often do not get money, so it's nice to be at least acknowledged. But, of course, it is not that important, especially if the recipient doesn't feel like they earned it.

And I just wanted to point out that I believe the only (?) git trailer used by github to link to github profiles is Co-authored-by. That's all.

**Comment by lima-limon-inc** [comment]
> Is it important? 🤷‍♂️ In fact, I think that I don't deserve much credit here, so feel free to squash the commits into one and drop the Suggested-by.

Hello @ulm. Like you suggested, I squashed the two commits into 1. Thank you for all the feedback!

File: app-emacs/projectile/projectile-2.8.0-r1.ebuild [hunk 2]
```diff
@@ -25,3 +25,12 @@ src_test() {
 
 	elisp-test
 }
+
+pkg_postinst() {
+	#Descriptions for this packages' purpose were taken from Projectile's
+	#home page https://docs.projectile.mx/projectile/usage.html
+	optfeature_header "Performance can be gained in ${PN} by installing"
+	optfeature_header "the following packages:"
```
Comment (ulm) [inline app-emacs/projectile/projectile-2.8.0-r1.ebuild:33]: `optfeature_header` doesn't print anything by itself, but only sets up the header to be printed by the following `optfeature` commands. So when it is called twice consecutively, the first call will be ignored.
```suggestion
	optfeature_header "You can improve performance with the following packages:"
```

**Review by ulm — CHANGES_REQUESTED** [review]

File: app-emacs/projectile/projectile-2.8.0-r1.ebuild (hunk 2 above)
Comment (lima-limon-inc) [reply app-emacs/projectile/projectile-2.8.0-r1.ebuild]: Hi @ulm, I was not aware of optfeature_header functionality, good to know! 
Thank you very much for the feedback. 
I submitted a new commit with your suggestion

**Comment by ulm** [comment]
Merged. Thank you!

**Pull request closed without merging**